### ISSUE TITLE
Add CSV loader

### ISSUE THREAD

**Issue by Priyamakeshwari** [Priyamakeshwari/TeachGPT#5]
Add CSV loader in builder to allow reading CSV files

**Comment by Priyamakeshwari** [comment]
yes . We already have an pdf loader to read all the pdf files . your issue is to add an csv loader  to read the csv files . It is similar to the pdf loader which we have already put in our code

**Comment by VishalS-HK** [comment]
So should it be inside load_docs itself or do I have the liberty to create a new function?

**Comment by VishalS-HK** [comment]
I think I have an idea. But it could take a while(a few days). Is that fine?

**Comment by Priyamakeshwari** [comment]
It must be inside the load _docs .

**Comment by VishalS-HK** [comment]
Cool. So could I be assigned this issue?

**Comment by Priyamakeshwari** [comment]
Sure ! Go Ahead .
